source/tests/pd/model/test_permutation_denoise.py

```
@@ -0,0 +1,109 @@
+model_dpa1.pop("fitting_net")
+model_dpa2["type_map"] = ["O", "H", "B", "MASKED_TOKEN"]
+model_dpa2.pop("fitting_net")
+model_hybrid["type_map"] = ["O", "H", "B", "MASKED_TOKEN"]
+model_hybrid.pop("fitting_net")
+
+
+class PermutationDenoiseTest:
+    def test(
+        self,
+    ) -> None:
+        generator = get_generator(GLOBAL_SEED)
```

Ruff (0.8.0)

Local variable `generator` is assigned to but never used

Remove assignment to unused variable `generator`

(F841)